### PR TITLE
basic checkout email authentication done

### PR DESCRIPTION
**Pull request by chin2yere**

### CODE REVIEW

File: Petits-debuts-api/server.js [hunk 1]
```diff
@@ -324,8 +335,71 @@ app.post("/money/update", async (req, res) => {
...
+cron.schedule("0 0 * * *", function () {
+  const isServiceOrder = (obj) => {
+    const firstValue = obj[Object.keys(obj)[0]];
+    if (typeof firstValue !== "number") {
+      return true;
+    } else {
+      return false;
+    }
 
+    //return firstValue instanceof Date;
```
Comment (zderick) [inline Petits-debuts-api/server.js]: looks like we have some unnecessary comment here

File: Petits-debuts-api/server.js [hunk 2]
```diff
@@ -324,8 +335,71 @@ app.post("/money/update", async (req, res) => {
...
+              sendEmail(msg);
+            }
+          }
+        }
+      }
+    } catch {}
+  };
+  getAllServiceOrders();
+
+  console.log("running a task every minute");
```
Comment (zderick) [inline Petits-debuts-api/server.js]: we can remove this now? @chin2yere

File: Petits-debuts-api/server.js [hunk 3]
```diff
@@ -324,8 +335,71 @@ app.post("/money/update", async (req, res) => {
...
+  const getAllServiceOrders = async () => {
+    try {
+      const serviceOrder = await Order.findAll({
+        include: [{ model: User, as: "user" }],
+        order: [["createdAt", "DESC"]],
+      });
+      for (let i = 0; i < serviceOrder.length; i++) {
+        console.log("inside for loop");
+        if (isServiceOrder(serviceOrder[i].order)) {
+          console.log("inside if statement");
```
Comment (zderick) [inline Petits-debuts-api/server.js]: Once we're ready, we can take these console logs out

File: Petits-debuts-api/server.js [hunk 4]
```diff
@@ -324,8 +335,71 @@ app.post("/money/update", async (req, res) => {
...
+            const month = (appointment.getUTCMonth() + 1)
+              .toString()
+              .padStart(2, "0");
+
+            if (appointment > today) {
+              const product = await Product.findOne({
+                where: { id: key },
+              });
+              const msg = {
+                to: "chinyereoffor@meta.com", // Change to your recipient
```
Comment (zderick) [inline Petits-debuts-api/server.js:475]: This is used for testing only, right? The recipient can change based on the order, and should be from the Service Order

Comment (chin2yere) [reply Petits-debuts-api/server.js:475]: yes, but for now, i used my meta email just so we can demonstrate that it works
I could change it to the appropriate value after presentation.

**Comment by chin2yere** [comment]
Node-cron schedule send done
Basic email checkout feature implemented
Users can now see their business page with all associated products
Users can create new businesses and/or products
recent orders page completed
previous feedback integrated
readme updated ---halfway

**Pull request merged**